test/collections.js

```diff
@@ -563,6 +563,8 @@
     equal(3, _.max([1, 2, 3, 'test']), 'Finds correct max in array starting with num and containing a NaN');
     equal(3, _.max(['test', 1, 2, 3]), 'Finds correct max in array starting with NaN');
 
+    equal([3, 6], _.map([[1, 2, 3], [4, 5, 6]], _.max), 'Finds correct max in array when mapping through multiple arrays');
```

Comment:
What course of action do I need to take in order to make this content high value enough to be merged?

Maybe I misunderstood something, I am new to contributing to open source; Do I need to refactor my code or my commit history to meet Underscore standards?